src/main/resources/templates/MailService/contact.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+</head>
+<body style='background: #fff;font-family: Arial, sans-serif;font-size: 11pt;'>
+<p>Beste meneer, mevrouw,</p>
+
+<p>{contactMessage}</p>
```

Comment:
Misschien beginnen met : 

Er is iemand die contact met u wilt opnemen. De persoon heeft de volgende melding voor u : {contactMessage}